packages/hagrid/hagrid/orchestra.py

```diff
@@ -272,6 +272,7 @@ def deploy_to_python(
         "n_consumers": n_consumers,
         "create_producer": create_producer,
         "association_request_auto_approval": association_request_auto_approval,
+        "background_tasks": True,
```

Comment:
we need to propagate this `background_tasks` flag to orchestra.launch

Reply:
@khoaguin I think we missed addressing this comment. Need to integrate background_tasks to def launch and then propogate it to other deployment types.

Reply:
Thank you very much @shubham3121
Integrating this in this commit https://github.com/OpenMined/PySyft/pull/8795/commits/8182a1480a6dfcf3f56568428b35aa644e475807